### ISSUE TITLE
Android : oWebViewInterface not defined

### ISSUE THREAD

**Issue by tdhman** [shripalsoni04/nativescript-webview-interface#40]
I have a NativeScript application containing a webview that loads the static html/js resource from local asset resource. In order to communicate between the js code and the application, I initialize the webview interface when the application's page is loaded. However, sometime I got this error on the function `getAndroidJSInterface` when trying to navigate to another page.

```
TypeError: Cannot read property '_onWebViewEvent' of undefined
```

The error occurred only in Android at the code inside the function `getAndroidJSInterface `

```
function getAndroidJSInterface(oWebViewInterface) {
  var AndroidWebViewInterface = com.shripalsoni.natiescriptwebviewinterface.WebViewInterface.extend({
    /**
     * On call from webView to android, this function is called from handleEventFromWebView method of WebViewInerface class
     */
    onWebViewEvent: function onWebViewEvent(webViewId, eventName, jsonData) {
      // getting webviewInterface object by webViewId from static map.
      var oWebViewInterface = getWebViewIntefaceObjByWebViewId(webViewId);

      oWebViewInterface._onWebViewEvent(eventName, jsonData); // oWebViewInterface not defined here
    }
  }); // creating androidWebViewInterface with unique web-view id.

  return new AndroidWebViewInterface(new java.lang.String('' + oWebViewInterface.id));
}
```

Any suggestion for this bug?
Thanks.

**Comment by tdhman** [comment]
Just found out that the is a PR for this bug #29

**Comment by hettiger** [comment]
@shripalsoni04 can we get PR #29 get merged eventually? Are there any questions?

**Comment by shripalsoni04** [comment]
PR #29 is merged and fix is released with version 1.4.3.
